fvm_actor_utils/src/util.rs

```diff
@@ -27,25 +28,39 @@ type ActorResult<T> = std::result::Result<T, ActorError>;
 /// It provides higher level utilities than raw syscalls for actors to use to interact with the
 /// IPLD layer and the FVM runtime (e.g. messaging other actors)
 #[derive(Clone, Debug)]
-pub struct ActorRuntime<S: Syscalls, BS: Blockstore> {
+pub struct ActorRuntime<S: Syscalls + Clone, BS: Blockstore + Clone> {
```

Comment:
I don't like having the `Clone` trait bound being spammed everywhere. Going to try changing the `Token` handle in `frc46_token` to use a mutable borrow instead of taking ownership, as it does for the token state. They are typically created together and passed to the `Token` so it shouldn't have a large impact on the code and allows me to avoid the `clone` call that caused all this.

Reply:
Yes, please do try that. This could be awkward for built-in actors too